core-bundle/contao/dca/tl_image_size.php

```diff
@@ -194,15 +189,16 @@
 		),
 		'preserveMetadata' => array
 		(
-			'inputType'               => 'checkbox',
+			'inputType'               => 'radio',
+			'options_callback'        => array('tl_image_size', 'getMetadataOptions'),
```

Comment:
Why not use a static options array and something like `$GLOBALS['TL_LANG']['tl_image_size']['preserveMetadataOptions']` as `reference`?

Reply:
Changed in 5d11fb04055a68de7952e68f6f429be8427209e7.